### PR TITLE
Docker image

### PR DESCRIPTION
**Pull request by misli**
Hi,
I've created this simple Dockerfile to build docker image with **sentrylogs**.
I've also created an automated build on Docker Hub ([misli/sentrylogs](https://hub.docker.com/r/misli/sentrylogs/)).
If You merge this pull request, I'll change the git repo to build directly from [mdgart/sentrylogs](https://github.com/mdgart/sentrylogs) or you may consider creating your own automated build.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM python:3-alpine
+
+MAINTAINER Jakub Dorňák <jakub.dornak@misli.cz>
```
Comment (bittner) [inline Dockerfile]: `MAINTAINER` is deprecated [according to the Docker docs](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated). You should use `LABEL maintainer="..."` instead.

**Comment by bittner** [comment]
NOTE: The build is failing because `flake8` crawls into the `.tox/` folder. Not your fault.

**Comment by mdgart** [comment]
Anything we can do to fix build?

**Comment by bittner** [comment]
@mdgart I've already fixed the root of the failure in PR #23. The problem is unrelated to @misli's changes.

We can rebase and merge the changes, but I'd prefer to have the `MAINTAINER` tag fixed first. Can you do that, @misli, and also rebase your changes, so we can confirm the build is passing?

**Comment by misli** [comment]
Done.

**Comment by bittner** [comment]
@mdgart Can you please decide whether you want the Dockerfile in your repository, and press the merge button? @misli also asked a question about the DockerHub repository that needs to be answered. Thank you!

**Pull request merged**